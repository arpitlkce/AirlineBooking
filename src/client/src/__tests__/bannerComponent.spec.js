import React from 'react';
import renderer from 'react-test-renderer';
import BannerComponent from '../bannerComponent';

describe('BannerComponent Component', () => {
    it('should render correctly in  mode', () => {
        const component = renderer.create(<BannerComponent />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
