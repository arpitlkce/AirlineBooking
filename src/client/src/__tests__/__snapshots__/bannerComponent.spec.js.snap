// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerComponent Component should render correctly in  mode 1`] = `
<nav
  className="navbar navbar-expand-md navbar-light bg-light"
>
  <a
    className="navbar-brand"
    href="/"
  >
    Go Fly
  </a>
  <ul
    className="mar-auto navbar-nav"
  >
    <li
      className="nav-item"
    >
      <h3>
         Welcome to Go Fly 
      </h3>
    </li>
  </ul>
  <button
    aria-label={null}
    className="btn btn-light"
    onClick={[Function]}
  >
    Log Out
  </button>
</nav>
`;
